### ISSUE TITLE
2factor auth

### ISSUE THREAD

**Issue by ThomasWaldmann** [nsupdate-info/nsupdate.info#47]
ThomasWaldmann wrote:
later add 2factor auth, TOTP, QR-Code rendering.

**Comment by stevleibelt** [comment]
Hey there,

How is the status of this issue? Is there anything that prevents us/you from implementing this feature?

Best regards,
Stev

**Comment by ThomasWaldmann** [comment]
No, but updating the code to recent django (see PRs) and keeping out scammmers / letting in legitimate users has higher prio right now.
